examples/node-app-mssql-ts/package.json

```
@@ -12,12 +12,12 @@
     "rollback": "../../bin/run-dev.sh migrate-rollback"
   },
   "dependencies": {
-    "@leapfrogtechnology/sync-db": "^1.0.0-beta.10",
-    "knex": "^0.95.2",
+    "@leapfrogtechnology/sync-db": "1.2.1",
```

This should change to `2.0.0` after we merge this change? So this means `sync-db` works without any changes and just upgrading the knex peer dependency in the example file?

```suggestion
    "@leapfrogtechnology/sync-db": "1.2.1",
```

---

I used the build zip created by `yarn pack` to test my changes. I have reverted that here and instead upgraded it to the latest version. Yes, will need to update this to v2.0.0 after the package is released